contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1801,7 +1836,7 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     {
         transactionContext.ovmL1TXORIGIN = address(0);
         transactionContext.ovmTIMESTAMP = 0;
-        transactionContext.ovmNUMBER = 0;
+        transactionContext.ovmNUMBER = DEFAULT_UINT256;
```

Comment:
It kind of expands the scope, but I can see an argument fo applying the change to the other values.

Reply:
It does seem a bit weird for this to be applied to `ovmNUMBER` and not to other values. If we *do* decide to keep the scope limited, we should at least add a comment explaining exactly why we do this for `ovmNUMBER` and not elsewhere.